snapshots/kremlib/Makefile

```diff
@@ -80,7 +82,7 @@ unit-tests: libhacl.so
 	-I . -I ../snapshots/kremlib -I ../../test/test-files/ -I $(TWEETNACL_HOME) \
 	$(TWEETNACL_HOME)/tweetnacl.c ../../test/test-files/hacl_test_utils.c \
 	../../test/test-files/unit_tests.c libhacl.so -o unit_tests.exe
-	./unit_tests.exe
+	LD_LIBRARY_PATH=. PATH=. DYLD_LIBRARY_PATH=. ./unit_tests.exe
```

Comment:
Nice catch, I think this should be done with `unit_tests32.exe` too

Reply:
Yep, this needs to go there as well. I didn't run them. But I'll add it.